cli/tests/testdata/check_js_points_to_ts.bar.ts

```diff
@@ -0,0 +1,3 @@
+export function bar(): string {
```

Comment:
Would you be able to move these under `testdata/run/check_js_points_to_ts/...`? This top level folder has too many files and I want to clean it up one day. Thanks!

Reply:
Done.